src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,13 @@
 
 public class Application {
     public static void main(String[] args) {
```

Comment:
In the main method, you should create only one instance of the Lottery class, and after that use the method `getRandomBall()` in loop to get balls with a random number and color.